docs/source/additional_features.rst

```diff
@@ -6,8 +6,8 @@ of the previous sections.
 
 .. _attrs_package:
 
-The attrs package
-*****************
+The attrs package and dataclasses
+*********************************
```

Comment:
Maybe create a separate section for dataclasses? New Python users might not be familiar with attrs but may know dataclasses since they are in the standard library, so I think that it makes sense to give them both equal weight, so that it's possible to learn about dataclass support without having to first read about attrs.